lua/cfc_loadouts/client/cl_cfc_functions.lua

```diff
@@ -1,32 +1,47 @@
 -- Functions
+CFCLoadouts = { }
 
-CFCLoadouts = {}
-
-function CFCLoadouts.createWeaponIcon ( X, Y, ent )
+function CFCLoadouts.createWeaponIcon( X, Y, ent )
     local weaponIcon = vgui.Create( "ContentIcon", weaponCatPanel )
     weaponIcon:SetPos( X, Y )
     weaponIcon:SetName( ent.PrintName or ent.ClassName )
     weaponIcon:SetSpawnName( ent.ClassName )
     weaponIcon:SetMaterial( "entities/" .. ent.ClassName .. ".png" )
     weaponIcon.weaponClass = ent.ClassName
-
     weaponIcon.selectionShape = vgui.Create( "DShape", weaponIcon )
     weaponIcon.selectionShape:SetType( "Rect" ) -- This is the only type it can be
-    weaponIcon.selectionShape:SetPos( 5, 5 )
-    weaponIcon.selectionShape:SetColor( Color( 255, 0, 255, 200 ) )
+    weaponIcon.selectionShape:SetPos( 3, 3 )
+    weaponIcon.selectionShape:SetColor( Color( 10, 10, 10, 240 ) )
```

Comment:
Making new `Color`, `Vector`, `Angle`, etc. objects is markedly expensive.
It won't be an issue here, but extracting it to something like `CFCLoadouts.Colors` table would make it more efficient and more easily configurable in the future.

Same comment goes for the rest of the colors

Reply:
Normally I'd do that yeah, in this case however the entire panel is only drawn once and wouldn't get called after that so I didn't.